src/components/AmountFilter.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { Button, Popover, InputLabel, Grid } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import RemoveIcon from '@material-ui/icons/Remove';
+import AmountTextField from './AmountTextField';
+
+const useStyles = makeStyles((theme) => ({
+  buttonStyle: {
+    margin: theme.spacing(1),
+    maxWidth: 120,
+    height: 57,
+  },
+  labelStyle: { paddingLeft: theme.spacing(1) },
+  dashStyle: { margin: '45px auto auto' },
+  filterStyle: { margin: '10px 0' },
+}));
+
+export default function AmountFilter() {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = useState(null);
```

Comment:
What is anchorEl and how is it used?

Reply:
`anchorEl` is the HTML element that the popover uses to set its position. In the `handleClick` function I set the `anchorEl` to the `event.currentTarget` (which is the Amount button). Maybe a better name would be `anchorElement` to make it less confusing?

Reply:
I think the current name is fine (el -> element is a logical implication), I was just wondering how it's used.